keyboards/tominabox1/le_chiffre/info.json

```diff
@@ -1,10 +1,82 @@
 {
-  "keyboard_name": "Le Chiffre",
-  "manufacturer": "tominabox1",
-  "url": "",
-  "maintainer": "tominabox1",
-  "usb": {
-    "vid": "0x7431",
-    "pid": "0xD645"
-  }
+    "keyboard_name": "Le Chiffre",
+    "manufacturer": "tominabox1",
+    "url": "",
+    "maintainer": "tominabox1",
+    "features": {
+        "bootmagic": true,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true,
+        "rgb_matrix": true
```

Comment:
Since OLEDs are enabled at the keyboard level, could you place the current OLED code in a `le_chiffre.c` file and remove it from the keymaps

Reply:
Ah, I see, thanks for the consideration. A future PR is a fine approach.